onadata/libs/utils/image_tools.py

```diff
@@ -56,22 +53,19 @@ def _save_thumbnails(image, path, size, suffix):
 
 def resize(filename):
     default_storage = get_storage_class()()
-    path = default_storage.url(filename)
-    req = requests.get(path)
-
-    if req.status_code == 200:
-        try:
-            im = StringIO(text(req.content))
-            image = Image.open(im)
-            conf = settings.THUMB_CONF
-
-            for key in settings.THUMB_ORDER:
-                _save_thumbnails(
-                    image, filename,
-                    conf[key]['size'],
-                    conf[key]['suffix'])
-        except IOError:
-            raise Exception("The image file couldn't be identified")
+
+    try:
+        path = default_storage.open(filename)
```

Comment:
The variable `path` could be misleading when I `default_storage.open(filename)` returns a file object. Could this be addressed as
```python

with default_storage.open(filename) as image_file:
    image = Image.open(image_file)
    ...
```